routers/routes/routes.go

```diff
@@ -853,6 +869,29 @@ func RegisterRoutes(m *macaron.Macaron) {
 			m.Get("/milestones", reqRepoIssuesOrPullsReader, repo.Milestones)
 		}, context.RepoRef())
 
+		m.Group("/projects", func() {
+
+			m.Get("", repo.Projects)
+			m.Get("/new", repo.NewProject)
+			m.Post("/new", bindIgnErr(auth.CreateProjectForm{}), repo.NewRepoProjectPost)
+			m.Group("/:id", func() {
+				m.Get("", repo.ViewProject)
+				m.Post("", bindIgnErr(auth.EditProjectBoardTitleForm{}), repo.AddBoardToProjectPost)
+				m.Post("/delete", repo.DeleteProject)           //ToDo: this has to be DELETE
```

Comment:
On routes.go Post for DELETE is fine.

```suggestion
				m.Post("/delete", repo.DeleteProject)
```